test/data/failExec.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
```

Comment:
Do not use `bash`, use `sh` for better portability. Also, use `#!/usr/bin/env sh` so that we do not depend on shell being in /bin e.g. in case of nix.